tests/src/ExistingSite/IntegratedPermissionsTest.php

```diff
@@ -3,55 +3,18 @@
 namespace Drupal\Tests\acquia_cms\Functional;
```

Comment:
@apathak18 `namespace Drupal\Tests\acquia_cms\Functional;` needs to be updated to `namespace Drupal\Tests\acquia_cms\ExistingSite;`